components/shared/UserMenu.tsx

```diff
@@ -9,12 +9,22 @@ import { resetStoreAction } from '@/store/store'
 import Image from 'next/image'
 import { useDispatch, useSelector } from 'react-redux'
 import Transition from './Transition'
-
+import { getAuthState } from '@/store/authSlice'
+import { CopyTextWithToolTip } from '@/components/CopyTextWithToolTip'
+import { getKanmonConnectSlice } from '@/store/kanmonConnectSlice'
 export const UserMenu = ({ align }: { align: string }) => {
   const [dropdownOpen, setDropdownOpen] = useState(false)
   const dispatch = useDispatch()
   const [clickCounter, setClickCounter] = useState(0)
   const { buttonBgColor } = useSelector(getCustomizationState)
+  const authState = useSelector(getAuthState)
+  const kanmonConnectState = useSelector(getKanmonConnectSlice)
+
+  const shouldDisplayDevToolIds =
```

Comment:
do we need this conditional? can't we just always display them? demo-platform is expected to be used in non-prod environments anyway.